editoast/src/models/mod.rs

```diff
@@ -20,6 +20,10 @@ pub use study::{Study, StudyWithScenarios};
 pub use timetable::{Timetable, TimetableWithSchedules};
 pub use train_schedule::TrainSchedule;
 
+pub trait Id {
```

Comment:
We can rename it and implement it automatically for objects that already implement the `diesel::Identifiable<Id = i64>`

```suggestion
pub trait Identifiable {
```

```rust
impl<T: Identifiable<Id = i64> + Clone> Id for T {
    fn get_id(&self) -> i64 {
        self.clone().id()
    }
}
```